### PR TITLE
github: Run fuzzing nightly

### PR DESCRIPTION
**Pull request by jepio**
## Description

Add a trigger to the fuzzing workflow that runs it nightly. The fuzzing runs will need to be periodically reviewed to see if there are any failures.

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [x] All unit tests are passing.
- [x] I have merged the latest `main` branch prior to this PR submission.
- [x] I submitted this PR against the `main` branch.

### CODE REVIEW

File: .github/workflows/fuzzing.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ name: Fuzzer execution
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * *'
```
Comment (AhmedBM) [inline .github/workflows/fuzzing.yml:6]: Can we also leave a comment with the frequency, similar to what we have here https://github.com/Azure/azure-osconfig/blob/d29a9ac7bd7569abef2de58de4c1ebd843370a61/.github/workflows/insiders-fast.yml#L6

**Comment by AhmedBM** [comment]
LGTM just some comments on the frequency of the workflow

**Review by MariusNi — APPROVED** [review]
:shipit:

File: .github/workflows/fuzzing.yml (hunk 1 above)
Comment (jepio) [reply .github/workflows/fuzzing.yml]: Added a comment

**Review by robertschaedler3 — APPROVED** [review]

**Pull request merged**